src/block-management/block-toolbar.js

```diff
@@ -37,34 +37,36 @@ export class BlockToolbar extends Component<PropsType> {
 
 		return (
 			<View style={ styles.container }>
-				<Toolbar>
-					<ToolbarButton
-						label={ __( 'Add block' ) }
-						icon="insert"
-						onClick={ onInsertClick }
-					/>
-					<ToolbarButton
-						label={ __( 'Undo' ) }
-						icon="undo"
-						isDisabled={ ! hasUndo }
-						onClick={ undo }
-					/>
-					<ToolbarButton
-						label={ __( 'Redo' ) }
-						icon="redo"
-						isDisabled={ ! hasRedo }
-						onClick={ redo }
-					/>
-				</Toolbar>
-				{ showKeyboardHideButton && ( <Toolbar>
-					<ToolbarButton
-						label={ __( 'Keyboard hide' ) }
-						icon="arrow-down"
-						onClick={ onKeyboardHide }
-					/>
-				</Toolbar> ) }
-				<BlockControls.Slot />
-				<BlockFormatControls.Slot />
+				<ScrollView horizontal={ true } showsHorizontalScrollIndicator={ false } keyboardShouldPersistTaps={ 'always' } alwaysBounceHorizontal={ false } >
```

Comment:
Hey @pinarol , is there a chance that we break this line into multiple lines?
I think it would be much easier for review :)

Reply:
sure